### DISCUSSION TITLE
COULD NOT PROCEED SINGING UP TO STUDENT BACKAGE

### DISCUSSION THREAD

**Discussion by ohaa99** [community/community#87453] [GitHub Education] [answered]
![image](https://github.com/community/community/assets/28456236/c3820192-ad6d-4bdf-8bbc-68b9b87cb5ef)
![Screenshot 2024-01-15 125143](https://github.com/community/community/assets/28456236/a47ffb82-38dc-40bb-b717-5d450afe9d8d)
Dear GitHub Community,

I hope this message finds you well. I am currently facing an urgent issue regarding the submission of proof for my current academic status. Despite my efforts and adherence to the provided guidelines, I am encountering difficulties when attempting to upload the required photo.

The challenge lies in the fact that the platform is not recognizing the file, and I'm unable to directly upload a photo using the camera. I've ensured that my camera settings are correct, but it seems there might be an issue with the file format or upload mechanism.

I am seeking urgent assistance from the community to help troubleshoot this problem. If anyone has encountered a similar issue or has insights into the correct file format or other potential solutions, your guidance would be greatly appreciated.

I am eager to complete the verification process promptly and appreciate any prompt assistance or advice the community can provide.

Thank you for your time and support.

Best regards,

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @ohaa99 , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by afeuerstein** [comment]
Seems to be the same problem as here: https://github.com/orgs/community/discussions/88401#discussioncomment-8148320
(#88401)

**Comment by mishmanners** [comment, marked as answer]
The same is resolved here: https://github.com/orgs/community/discussions/88401#discussioncomment-8142514 and here: https://github.com/orgs/community/discussions/88446#discussioncomment-8148738

Please see these. The TL;DR:
- Refresh the page or clear your browser's cache and cookies
- Check for browser extensions that could be interfering with the JavaScript code and disable them
- Try a different browser
- Ensure your file is less than 150kB
